training/distributed_training/pytorch/model_parallel/gpt2/smp-train-gpt-simple.ipynb

```diff
@@ -0,0 +1,716 @@
+{
```

Comment:
<p>Nit - "requires a small memory" sounds odd to me - maybe "requires a small amount of memory", or be more specific, like "typically requires no more than a few MB of memory"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3034/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>Thanks for suggestions. Using "requires a small amount of memory".</p><p> </p>